projects/exui-common-lib/src/lib/components/selected-case/selected-case.component.ts

```diff
@@ -56,10 +64,9 @@ export class SelectedCaseComponent implements OnInit {
         }
       }
     });
-    return canRemove;
+    return removeUserFromCaseToggleOn && canRemove;
```

Comment:
This leads to race conditions, as you're relying on the fact that the subscription finished before you reach this line. This method should return an Observable<boolean> itself, and use the two observables you subscribe to now to combine them to create the final value, and return that in an Observable.

Reply:
moved the flag to webapp as per other comments.